bin/generate-form-docs

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+'use strict';
+const { stripIndents } = require('common-tags');
+const fs = require('fs');
+const path = require('path');
+
+function generateFormDocs(formBuilder, directory) {
+    const form = formBuilder();
+
+    const fieldSummary = Object.values(form.allFields)
+        .map(function (field) {
+            return stripIndents`
+                ## ${field.name}
+
+                | Rule           | Message              |
+                | -------------- | -------------------- |
```

Comment:
If you wanted to be a mega nerd here you could find the longest `item.type` in the list and make the Rule dashes here long enough to accomodate it, or does Markdown do that for you regardless?

Reply:
Update, never mind, looks like it does!

Reply:
Yep, plus prettier can tidy it all up afterwards too